ffi/diplomat/c/examples/fixeddecimal/Makefile

```diff
@@ -13,11 +13,12 @@ $(ALL_RUST):
 $(ALL_HEADERS):
 
 
-../../../../../target/debug/libicu_capi_staticlib.a: $(ALL_RUST)
-	cargo build -p icu_capi_staticlib
+../../../../../target/debug/libicu_capi_staticlib_test.a: $(ALL_RUST)
+	cargo build -p icu_capi_staticlib --features provider_test
+	mv ../../../../../target/debug/libicu_capi_staticlib.a ../../../../../target/debug/libicu_capi_staticlib_test.a
```

Comment:
nit: cp probably

Reply:
I want to keep the non-suffixed name for the default feature compilation. This should make the makefiles trip up less.

Reply:
Makes sense!